src/definitions/modules/progress.js

```diff
@@ -260,16 +260,13 @@
 
                 get: {
                     text: function (templateText, index) {
-                        if (!index) {
-                            index = 0;
-                        }
-
                         var
-                            value   = module.get.value(index),
+                            index_  = index || 0,
+                            value   = module.get.value(index_),
                             total   = module.get.total(),
                             percent = animating
-                                ? module.get.displayPercent(index)
-                                : module.get.percent(index),
+                                ? module.get.displayPercent(index_)
```

Comment:
We have reduced that in the previous PRs...now you put in again?

Reply:
see https://github.com/fomantic/Fomantic-UI/pull/2589#discussion_r1043860140, when I was splitting PRs, I reverted this on top of the commits, thus the change was merged...

I am fine to keep this few lines for 2.10.x as agreed before, so reverted here.

Reply:
I actually liked the changes now, especially for this codeblock (the mix using index_ is ugly)
By dropping IE11 we can simply remove the "if (!index)" block in the beginning.

Same for the other code (if !action), i dont want to change code again and again. So please dont change them agai. They are working fine as they are and satify the linter :)

Reply:
perfect, revert reverted :)